<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PicturesPainterClass</class>
 <widget class="QDialog" name="PicturesPainterClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PicturesPainter</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2,1">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4,1">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="styleSheet">
          <string notr="true">QGroupBox {
    border: 1px solid #DDD;
    border-radius: 4px;
    font: bold 14px;
    color: #666;
    background-color: rgb(202, 202, 202);
    padding: 10px; 
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left: 10px;
    padding: 0 5px;
}</string>
         </property>
         <property name="title">
          <string>绘图列表</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QListView" name="listView">
            <property name="styleSheet">
             <string notr="true">/* 基础列表视图 */
QListView {
    border: 1px solid #AAAAAA;
    border-radius: 3px;
    background: #FFFFFF;
    color: #444444;
    font-size: 20px;
    outline: 0; 
    padding: 2px;
}

/* 列表项通用样式 */
QListView::item {
    height: 28px;
    padding: 4px 8px;
    border-radius: 2px;
    margin: 1px 2px;
}

/* 悬停效果 */
QListView::item:hover {
    background: #F8F8F8;
    border: 1px solid #888888;
}

/* 选中状态 */
QListView::item:selected {
    background: #666666;
    color: #FFFFFF;
    border: 1px solid #555555;
}

/* 焦点状态 */
QListView::item:selected:active {
    background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #666666, 
        stop:1 #555555
    );
}

/* 禁用状态 */
QListView:disabled {
    border-color: #DDDDDD;
    color: #AAAAAA;
}

QListView::item:disabled {
    background: #F5F5F5;
    color: #AAAAAA;
}

/* 紧凑型变体 */
QListView.compact {
    font-size: 13px;
}

QListView.compact::item {
    height: 24px;
    padding: 2px 6px;
}

/* 滚动条样式 */
QScrollBar:vertical {
    width: 10px;
    background: #FFFFFF;
}

QScrollBar::handle:vertical {
    background: #CCCCCC;
    border-radius: 4px;
    min-height: 20px;
}

QScrollBar::handle:vertical:hover {
    background: #AAAAAA;
}

QScrollBar::add-line:vertical, 
QScrollBar::sub-line:vertical {
    height: 0px;
}

/* 强调型变体 */
QListView.emphasized::item:selected {
    background: #4A4A4A;
    border-color: #3A3A3A;
}

/* 交替行颜色 */
QListView::item:alternate {
    background: #FAFAFA;
}

/* 角部处理 */
QScrollBar #qt_scrollarea_corner {
    background: #FFFFFF;
}</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="styleSheet">
          <string notr="true">QGroupBox {
    border: 1px solid #DDD;
    border-radius: 4px;
    font: bold 14px;
    color: #666;
    background-color: #F8F8F8;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left: 10px;
    padding: 0 5px;
}</string>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="DrawLabel" name="label_imgDisplay">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QGroupBox {
    border: 1px solid #DDD;
    border-radius: 4px;
    font: bold 14px;
    color: #666;
    background-color: rgb(202, 202, 202);
    padding: 10px; 
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left: 10px;
    padding: 0 5px;
}</string>
         </property>
         <property name="title">
          <string>绘画设置</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout" stretch="2,2,1">
            <property name="sizeConstraint">
             <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
            </property>
            <item>
             <widget class="QPushButton" name="btn_set">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
    padding: 6px 14px;
    border: 2px solid #CCC;
    border-radius: 4px;
    background-color: white;
    color: #444;
    font-size: 20px; 
}

QPushButton:hover {
    border-color: #999;
    background-color: #F5F5F5;
}

QPushButton:pressed {
    border-color: #777;
    background-color: #EEE;
}</string>
              </property>
              <property name="text">
               <string>设置</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_draw">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
    padding: 6px 14px;
    border: 2px solid #CCC;
    border-radius: 4px;
    background-color: white;
    color: #444;
    font-size: 20px; 
}

QPushButton:hover {
    border-color: #999;
    background-color: #F5F5F5;
}

QPushButton:pressed {
    border-color: #777;
    background-color: #EEE;
}</string>
              </property>
              <property name="text">
               <string>绘画</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_clear">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
    padding: 6px 14px;
    border: 2px solid #CCC;
    border-radius: 4px;
    background-color: white;
    color: #444;
    font-size: 20px; 
}

QPushButton:hover {
    border-color: #999;
    background-color: #F5F5F5;
}

QPushButton:pressed {
    border-color: #777;
    background-color: #EEE;
}</string>
              </property>
              <property name="text">
               <string>清空</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,2">
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="styleSheet">
          <string notr="true">QGroupBox {
    border: 1px solid #DDD;
    border-radius: 4px;
    font: bold 14px;
    color: #666;
    background-color: rgb(161, 161, 161);
    padding: 2px; 
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left: 10px;
    padding: 0 5px;
}</string>
         </property>
         <property name="title">
          <string>详细信息</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="topMargin">
           <number>15</number>
          </property>
          <item row="0" column="0">
           <widget class="QTableView" name="tableView">
            <property name="styleSheet">
             <string notr="true">/* 基础表格视图 */
QTableView {
    border: 1px solid #AAAAAA;
    border-radius: 3px;
    background: #FFFFFF;
    color: #444444;
    font-size: 14px;
    outline: 0;
    gridline-color: #EEEEEE;  /* 网格线颜色 */
    alternate-background-color: #FAFAFA;  /* 交替行背景 */
    selection-background-color: #666666;  /* 选中背景 */
    selection-color: #FFFFFF;  /* 选中文字 */
}

/* 表头系统 */
QHeaderView {
    background: #FFFFFF;
    border: none;
    border-bottom: 2px solid #DDDDDD;
}

QHeaderView::section {
    background: #F8F8F8;
    border: none;
    border-right: 1px solid #DDDDDD;
    padding: 8px 12px;
    color: #666666;
    font-weight: 500;
    qproperty-defaultAlignment: AlignLeft|AlignVCenter;
}

QHeaderView::section:last-child {
    border-right: none;
}

/* 排序指示器 */
QHeaderView::down-arrow {
    image: url(:/icons/sort_desc_gray.svg);
    margin-right: 4px;
}

QHeaderView::up-arrow {
    image: url(:/icons/sort_asc_gray.svg);
    margin-right: 4px;
}

/* 单元格样式 */
QTableView::item {
    padding: 6px 12px;
    border: 1px solid transparent;
    min-height: 32px;
}

/* 悬停效果 */
QTableView::item:hover {
    background: #F8F8F8;
    border-color: #888888;
}

/* 选中状态 */
QTableView::item:selected {
    border-color: #555555;
}

/* 焦点单元格 */
QTableView::item:focus {
    border: 1px solid #888888;
    background: #FFFFFF;
}

/* 禁用状态 */
QTableView:disabled {
    border-color: #DDDDDD;
    color: #AAAAAA;
}

QTableView::item:disabled {
    background: #F5F5F5;
    color: #AAAAAA;
}

/* 紧凑型变体 */
QTableView.compact {
    font-size: 13px;
    gridline-color: #F0F0F0;
}

QTableView.compact::item {
    min-height: 28px;
    padding: 4px 8px;
}

/* 编辑状态 */
QTableView::item:edit {
    border: 1px solid #888888;
    background: #FFFFFF;
}

/* 滚动条系统 */
QScrollBar:horizontal {
    height: 10px;
    background: #FFFFFF;
}

QScrollBar::handle:horizontal {
    background: #CCCCCC;
    border-radius: 4px;
    min-width: 20px;
}

/* 角部处理 */
QTableView QTableCornerButton::section {
    background: #F8F8F8;
    border-bottom: 2px solid #DDDDDD;
}</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="styleSheet">
          <string notr="true">QGroupBox {
    border: 1px solid #DDD;
    border-radius: 4px;
    font: bold 14px;
    color: #666;
    background-color: rgb(161, 161, 161);
    padding: 2px; 
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left: 10px;
    padding: 0 5px;
}</string>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="pbtn_ok">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    padding: 6px 14px;
    border: 2px solid #CCC;
    border-radius: 4px;
    background-color: white;
    color: #444;
    font-size: 30px; 
}

QPushButton:hover {
    border-color: #999;
    background-color: #F5F5F5;
}

QPushButton:pressed {
    border-color: #777;
    background-color: #EEE;
}</string>
            </property>
            <property name="text">
             <string>确定</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbtn_exit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    padding: 6px 14px;
    border: 2px solid #CCC;
    border-radius: 4px;
    background-color: white;
    color: #444;
    font-size: 30px; 
}

QPushButton:hover {
    border-color: #999;
    background-color: #F5F5F5;
}

QPushButton:pressed {
    border-color: #777;
    background-color: #EEE;
}</string>
            </property>
            <property name="text">
             <string>退出</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbtn_setAspectRatio">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    padding: 6px 14px;
    border: 2px solid #CCC;
    border-radius: 4px;
    background-color: white;
    color: #444;
    font-size: 30px; 
}

QPushButton:hover {
    border-color: #999;
    background-color: #F5F5F5;
}

QPushButton:pressed {
    border-color: #777;
    background-color: #EEE;
}</string>
            </property>
            <property name="text">
             <string>设置纵横比</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbtn_openPicture">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    padding: 6px 14px;
    border: 2px solid #CCC;
    border-radius: 4px;
    background-color: white;
    color: #444;
    font-size: 30px; 
}

QPushButton:hover {
    border-color: #999;
    background-color: #F5F5F5;
}

QPushButton:pressed {
    border-color: #777;
    background-color: #EEE;
}</string>
            </property>
            <property name="text">
             <string>打开图片</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DrawLabel</class>
   <extends>QLabel</extends>
   <header>PicturesPainter.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
